dynamiqs/utils/operators.py

```diff
@@ -34,24 +34,21 @@
 def eye(*dims: int) -> Array:
     r"""Returns the identity operator.
 
-    If only a single dimension is provided, `eye` returns the identity operator
-    of corresponding dimension. If instead multiples dimensions are provided, `eye`
-    returns the identity operator of the composite Hilbert space given by the product
-    of all dimensions.
```

Comment:
I quite enjoyed these comments. This is kinda new behavior compared to QuTiP, so I wouldn't expect users to understand that just from the examples below. IMO we should keep it (same in other functions), unless you're really keen in removing it.

Reply:
Yes I agree, let me propose a mathematically precise formulation.